syntax = "proto2";
import "comm_db.proto";
option optimize_for = LITE_RUNTIME;

message DBBagGrid
{
	required uint32 id = 1;
	required uint32 num = 2;
	required uint32 used_count = 3;
	required uint32 expire_time = 4;
	optional DBItemBagua bagua = 5;
	optional DBItemPartnerFabao fabao = 6;
	optional DBItemRandomBox box = 7;
}

message DBHeadIcon
{
	required uint32 id = 1;
	required uint32 status = 2;
}

message DBTaskCount
{
	required uint32 id = 1;
	required uint32 num = 2;
}

message DBTask
{
	required uint32 id = 1;
	required uint32 status = 2;
	required uint32 accept_ts = 3;
	required uint32 accu_time = 4;
	repeated DBTaskCount progress = 5;
}

message ItemBuff
{
	required uint32 id = 1;
	required uint64 end_time = 2;
	required uint64 buff_state = 3;
}

message DBEquipEnchant
{
	required DBCommonRandAttr cur_attr = 1; //当前属性
	repeated DBCommonRandAttr rand_attr = 2; //随机出来的属性
}

message DBEquip
{
	required uint32 stair = 1; //阶数
	required uint32 star_lv = 2; //星数
	required uint32 star_exp = 3; //升星等级
	repeated DBEquipEnchant enchant = 4; //附魔
	repeated int32 inlay = 5; //镶嵌（-1：未开孔，0：未镶嵌，>0：宝石id）
}

message SkillDbData
{
	repeated SkillOneDbData all = 1; //
	required uint32 num = 2; //
}
message FuwenDb
{
	required uint32 id = 1;
	required uint32 lv = 2;
	required bool is_new = 3; //ture 新的
}
message SkillOneDbData
{
	required uint32 id = 1;
	required uint32 lv = 2;
	repeated FuwenDb fuwen = 3;
	repeated uint32 cur_fuwen = 4; //使用的符文
}

message FashionDbData
{
	required uint32 id = 1;
	required uint32 color = 2;
	required uint32 cd = 3; //剩余秒数 0：永久
	required bool isnew = 4;
	required uint32 colordown = 5; //下衣颜色
}

message DBGoods
{
	required uint32 goods_id = 1;
	required uint32 bought_num = 2;
}

message DBShopReset
{
	required uint32 next_day_time = 1;
	required uint32 next_week_time = 2;
	required uint32 next_month_time = 3;
}


message DbHorseData
{
	required uint32 id = 1;
	required uint64 cd = 2; //
	required bool isnew = 3;
	required uint32 step = 4; //铸灵
	required uint32 star = 5;//铸灵
}
message DbHorseCommonAttr
{
	required uint32 step = 1; //阶
	repeated uint32 attr = 2; //属性
	repeated uint32 attr_level = 3; //当前修灵次数
	required uint32 soul_step = 4; //铸灵等级
	required uint32 soul_star = 5; //当前可以升级的灵
	required uint32 power = 6; //总的战斗力
	repeated uint32 soul_num = 7; //当前已铸灵次数
	required bool soul_full = 8; //true铸灵已满
	required int32 fly = 9; //
	//required uint32 old = 10; //
}

message DBYuqidaoMai
{
	required uint32 mai_id = 1;
	required uint32 acupoint_id = 2;
	required uint32 fill_lv = 3;
}

message DBYuqidaoBreak
{
	required uint32 id = 1;
	repeated uint32 cur_val = 2;
	repeated uint32 new_val = 3;
	repeated uint32 new_addn = 4;
	required uint32 count = 5;
}

message DBBaguapaiCard
{
	required uint32 id = 1;
	required uint32 star = 2;
	repeated DBCommonRandAttr minor_attrs = 5;
	repeated DBCommonRandAttr minor_attrs_new = 6;
	repeated DBCommonRandAttr additional_attrs = 7;
}

message DBBaguapaiDress
{
	repeated DBBaguapaiCard cards = 1;
}

message GuoyuTypeDb
{
	required int32 guoyu_level = 1;
	required int32 cur_exp = 2;
	required int32 cur_task = 3; //0 没任务
	required int32 task_cd = 4; //0 超时 大于0 剩余cd  
	required int32 critical_num = 5; // 紧急剩余次数
	required int32 guoyu_num = 6; //国御剩余收益次数
	required int32 map = 7; //国御地图
	required int32 type = 8; //
	required uint32 rand_map = 9; //
	optional uint64 critical_cd = 10; // 
	optional uint64 critical_cd_refresh = 11; //
}
message ChengjieTypeDb
{
	required int32 level = 1;
	required int32 cur_exp = 2;
	required int32 num = 3; //剩余接任务次数
	required uint64 target = 4;
	required uint32 task = 5;
	required bool first_hit = 6;
}
// message ShangjinTaskAwardDb
// {
// 	required uint32 id = 1;
// 	required uint32 num = 2; //
// }
message ShangjinTaskTypeDb
{
	required uint32 id = 1;
	required uint32 quality = 2; //
//	repeated ShangjinTaskAwardDb award = 3; //
	required uint32 drop_id = 3;
	required uint32 reduce = 4; //
	required uint32 coin = 5; //
	required uint32 exp = 6; //
}
message ShangjinTypeDb
{
	required int32 level = 1;
	required uint32 cur_exp = 2;
	required uint32 num = 3; //剩余接任务次数
	required uint32 cur_task = 4; //
	repeated ShangjinTaskTypeDb task = 5; //
	required bool accept = 6;
	required uint32 free_refresh = 7; //免费刷新次数
}
message YaoshiDb
{
	required int32 cur_major = 1;  //MAJOR_TYPE
	required GuoyuTypeDb guoyu = 2;
	required ChengjieTypeDb chengjie = 3;
	required ShangjinTypeDb shangjin = 4;
	required uint32 change_num = 5;  //改变专精次数
}

message PvpRaidDb
{
	required uint32 oneday_win_num = 1;
	required uint32 cur_level_id = 2;
	required uint32 max_level_id = 3;
	required uint32 max_score = 4;
	required uint32 score = 5;
	required uint32 avaliable_reward_level = 6;
	repeated uint32 avaliable_box = 7;						
}

message DBDailyActivity
{
	required uint32 id = 1;
	required uint32 count = 2;
}
message DBChivalryActivity
{
	required uint32 id = 1;
	required uint32 val = 2;
}

message DBAwardQuestion
{
	required uint32 trun = 1; //第几轮 从1开始
	required uint64 next_open = 2; //
	required uint32 right = 3; //答对数
	required uint32 contin = 4; //连续答对
	required uint32 money = 6; //
	required uint32 exp = 7; //
	required uint32 timer = 8; //总耗时
	required uint32 question = 9; //题目
	required uint32 number = 10; //第几题
	required uint32 score = 11; //积分
}
message DBCommonQuestion
{
	required uint32 question = 1; //题目
	required uint32 contin = 2; //连续答对
	required uint32 right = 3; //答对数
	required uint32 money = 4; //
	required uint32 exp = 5; //
	required uint32 tip = 6; //剩余提示次数
	required uint32 help = 7; //剩余求助次数
	required bool btip = 8; //true 已经提示过
	required bool bhelp = 9; //true 已经求助过 
	required uint32 number = 10; //第几题
	required uint32 cd = 11; //活动剩余时间
	repeated uint32 answer = 12;
	repeated uint32 anstip = 13; //
	required uint64 next_open = 14; //
}

message DBPersonality
{
	required uint32 sex = 1;
	required uint32 birthday = 2;
	required string location = 3;
	repeated uint32 tags = 4;
	required string text_intro = 5;
	required string voice_intro = 6;
	required uint32 province = 7;
	required uint32 city = 8;
	required uint32 blood_type = 9;
}

message DBLiveSkill
{
	repeated uint32 level = 1; //当前技能等级
	repeated uint32 exp = 2; //当前技能经验
	repeated uint32 broken = 3; //当前技能突破等级
	repeated uint32 book = 4; //剩余使用技能书的次数
}

message DBPartnerSkill
{
	required uint32 skill_id = 1;
	required uint32 lv = 2;
	required bool lock = 3;
}

message DBPartnerAttr
{
	repeated DBPartnerSkill skills = 1; //技能列表
	repeated uint32 base_attr_id = 2; //基本属性id
	repeated uint32 base_attr_cur = 3; //基本属性当前值
	repeated uint32 base_attr_up = 4; //基本属性上限
	repeated uint32 detail_attr_id = 5; //详细属性id
	repeated uint32 detail_attr_cur = 6; //详细属性当前值
	required uint32 type = 7; //伙伴类型 0普通,1平凡,2天才
	required uint32 power_refresh = 8;
}

message DBPartner
{
	required uint64 uuid = 1; //唯一ID
	required uint32 partner_id = 2; //伙伴ID
	repeated DBAttr attrs = 5;
	required uint32 bind = 6;
	optional DBPartnerAttr attr_cur = 7;
	optional DBPartnerAttr attr_flash = 8;
	required uint32 relive_time = 9;
	repeated uint32 god_id = 10; //
	repeated uint32 god_lv = 11; //
	required uint32 strong_num = 12;
	optional DBCurPartnerFabao cur_fabao_info = 13;
	required string name = 14;
	required bool partner_rename_free = 15;  //伙伴免费改名一次
}

message DBTruck
{
	required uint64 truck_id = 1; //镖车
	required uint32 active_id = 2;
	required uint32 num_coin = 3; //粮草押镖次数
	required uint32 num_gold = 4; //财宝押镖次数
	required float pos_x = 5; //x坐标
	required float pos_z = 6; //z坐标
	required uint32 hp = 7;
	required uint32 scene_id = 8;
}

message DBZhenying
{
	required uint32 award_num = 1; //阵营对战收益次数
	required uint32 level = 2; //
	required uint32 exp = 3; //
	required uint32 task = 4; //当前任务ID
	required uint32 task_type = 5; //当前任务类型
	required uint32 task_num = 6; //当前任务进度
	required uint32 step = 7; //军阶
	required uint32 exp_day = 8; //
	required uint32 free_change = 9; //免费换阵营次数
	required uint32 kill_week = 10; //战场一周杀人
	required uint32 score_week = 11;
	required uint32 last_week = 12;//上次更新时间
	required uint32 week = 13; 
	required uint32 kill = 14;
	required uint32 death = 15;
	required uint32 help = 16;
	required uint32 score = 17;
	required bool one_award = 18; //首胜
	required uint64 fb_cd = 19;
	required uint64 exploit = 20;
	required uint64 history_exploit = 21;
	required bool daily_award = 22;  //军阶每日奖励
	required bool daily_step = 23;  //军阶每日奖励
	required uint32 task_award_state = 24;
	required uint32 step_lv = 25;
}

message DBAchievement
{
	required uint32 id = 1; //成就ID
	required uint32 star = 2; //当前已达成星数
	required uint32 progress = 3; //当前进度
	required uint32 state = 4; //
	required uint32 achieve_time = 5; //达成时间
}

message DBTitle
{
	required uint32 id = 1; //称号ID
	required uint32 state = 2;
	required uint32 expire_time = 3;
	required bool   is_new = 4;
	required uint32 active_time = 5;
}

message DBStrongGoal
{
	required uint32 id = 1; //目标ID
	required uint32 progress = 2; //当前进度
	required uint32 state = 3; //领奖状态
}

message DBStrongChapter
{
	required uint32 id = 1; //章节ID
	required uint32 progress = 2; //当前进度
	required uint32 state = 3; //领奖状态
}

message DBTowerInfo
{
	required uint32 reset_num = 1; //重置次数
	required uint32 cur_lv = 2; //当前层
	required uint32 cur_num = 3; //当前层次数
	required uint32 award_lv = 4;//
	required uint32 top_lv = 5; //最高层
	required uint32 rand_collect_num = 6; //随机宝箱采集次数 与冲塔无关
	required uint32 script_reward_num = 7; //剧情副本次数 与冲塔无关
}

message PlayerDBInfo
{
	required uint64 exp = 1; //经验
	required uint32 scene_id = 2; //场景id
	required float pos_x = 3; //x坐标
	required float pos_z = 4; //z坐标
	required float pos_y = 5; //y高度
	repeated DBBagGrid bag = 6; //背包
	required uint32 bag_grid_num = 7; //背包开启格子数
	required uint32 bag_unlock_num = 8; //背包解锁格子数
	required uint32 gold = 9; //元宝
	required uint32 bind_gold = 10; //绑定元宝
	required uint32 coin = 11; //银两
	required uint32 head_icon = 12; //头像
	repeated DBHeadIcon head_icon_list = 13; //头像列表
	repeated uint32 attr_id = 14; //属性ID
	repeated uint32 attr = 15; //属性值
	repeated DBTask task_list = 16; //正在进行的任务列表
	repeated uint32 task_finish = 17; //已经完成的任务列表
	required uint32 next_time_refresh_oneday_job = 18; //下次刷新每日事件的时间
	repeated ItemBuff item_buffs = 19;   //道具buff
	required uint32 cur_hp = 20; //当前血量
	required uint64 teamid = 21; //
	repeated uint32 raid_reward_id = 22;  //记录副本收益次数的副本ID
	repeated uint32 raid_reward_num = 23;  //记录副本收益次数
	repeated DBEquip equip_list = 24; //装备列表
	optional SkillDbData skill = 25;
	repeated FashionDbData fashion = 26;
	repeated uint32 color = 27;  //已解锁颜色
	repeated int32 color_isnew = 28;  //
	repeated DBGoods shop_goods = 29; //商城
	repeated DbHorseData horse_data = 30;
	required DbHorseCommonAttr horse_attr = 31;
	repeated DBYuqidaoMai yuqidao_mais = 32; //御气道经脉列表
	repeated DBYuqidaoBreak yuqidao_breaks = 33; //御气道冲脉列表
	required uint32 pk_type = 34;  //pk类型
	required uint32 murder_num = 35;  //杀戮值
	repeated DBBaguapaiDress baguapai_dress = 36; //八卦牌装备列表
	required uint32 baguapai_style = 37; //当前使用的八卦牌套
	optional YaoshiDb yaoshi = 38; //妖师客栈
	required uint32 qiecuo_invite_switch = 39;  //切磋邀请开关
	required uint32 team_invite_switch = 40;  //组队邀请开关
	required uint32 out_stuck_time = 41;  //脱离卡死CD时间
	optional PvpRaidDb pvp_3 = 42;  //PVP3数据
	optional PvpRaidDb pvp_5 = 43;	//pvp5数据
	repeated uint32 active_reward = 44;  //已领取的活跃度奖励
	repeated DBDailyActivity daily_activity = 45;  //日常活动
	repeated DBChivalryActivity chivalry_activity = 46;  //侠义活动
	optional DBAwardQuestion award_question = 47;
	optional DBCommonQuestion common_question = 48;
	required uint32 last_scene_id = 49;  //上一个野外场景
	optional DBAutoAddHp auto_add_hp = 50; //自动补血设置
	optional DBPersonality personality = 51; //个人信息
	optional DBLiveSkill live_skill = 52; //生活技能
	required uint32 leaveraid_sceneid = 53;//离开副本后要去的场景id
	required double ExitPointX = 54; //离开副本后要去的场景横坐标
	required double ExitPointY = 55; //离开副本后要去的场景高度坐标
	required double ExitPointZ = 56; //离开副本后要去的场景纵坐标
	required double FaceY = 57; //离开副本后朝向
	required bool NoviceRaid = 58; //新手副本是否完成的标志
	repeated DBPartner partner_list = 59; //伙伴列表
	repeated uint32 partner_dictionary = 60; //伙伴图鉴
	repeated uint64 partner_formation = 61; //伙伴阵型
	repeated uint64 partner_battle = 62; //伙伴出战
	required uint32 partner_recruit_junior_free_cd = 63; //低级招募免费时间
	required uint32 partner_recruit_junior_count = 64; //低级招募计数
	required uint32 partner_recruit_senior_free_cd = 65; //高级招募免费时间
	required uint32 partner_recruit_senior_count = 66; //高级招募计数
	required uint64 ReceiveGift = 67; //即将开启已领取最大奖励类型
	required float angle = 68;  //朝向
	optional DBTruck truck = 69; //押镖
	required uint32 charm_level = 70; //魅力
	required uint32 charm_total = 71; //魅力
	repeated uint32 weapon_color = 72;  //已解锁颜色
	repeated int32 weapon_color_isnew = 73;  //
	required uint32 task_chapter_reward = 74;  //任务章节奖励
	optional DBShopReset shop_reset = 75; //商城重置
	repeated uint32 partner_bond = 76; //伙伴羁绊
	repeated uint32 partner_bond_reward = 77; //伙伴羁绊奖励
	optional DBZhenying zhenying = 78; //阵营
	required uint32 server_level_break_count = 79; //服务器等级突破计数
	required uint32 server_level_break_notify = 80; //服务器等级突破通知
	required uint32 guild_battle_activity_time = 81; //所参加的帮会战活动开始时间
	repeated DBAchievement achievement_list = 82; //成就列表
	required bool partner_recruit_first = 83; //首次招募标志
	repeated DBTitle title_list = 84; //称号列表
	repeated DBHreoChallengeInfo hero_challenge_all_info = 85; //英雄挑战总信息
	optional DBPlayerDiGongXiuLianInfo di_gong_xiulian_info = 86; //地宫修炼
	required uint32 fishing_bait_id = 87;  //钓鱼鱼饵ID
	repeated DBStrongGoal strong_goals = 88; //变强目标
	repeated DBStrongChapter strong_chapters = 89; //变强章节
	required uint32 travel_round_num = 90;  //游历当前轮数
	required uint32 travel_round_count_out = 91;  //游历当前轮数是否算在今天轮数里
	required uint32 travel_task_num = 92;  //游历当前环数
	repeated DBPlayerLevelRewardInfo player_level_reward_all_info = 93; //玩家等级奖励所有信息
	required DBPlayerOnlineRewardInfo player_online_reward_info = 94; //玩家在线奖励信息
	optional DBPlayerSignInEveryDayInfo player_sign_in_info = 95;    //玩家签到信息
	repeated DBPlayerActiveRewardBackInfo player_reward_back_info = 96;    //玩家活动奖励找回信息
	optional DBPlayerYaoQianShuInfo  player_yao_qian_shu_info = 97;    //玩家摇钱树
	optional DBPlayerLoginRewardInfo  player_login_reward_info = 98;    //登录奖励信息
	repeated uint32 function_open = 99; //功能开启信息
	optional DBTowerInfo tower = 100; //冲塔
	repeated DBPlayerCifuRewardInfo ci_fu_reward_info = 101; //赐福奖励信息
	required uint32 fishing_reward_num = 102;  //钓鱼收益次数
	required uint32 guild_bonfire_reward_time = 103;  //帮会篝火累计收益时间
	required uint32 guild_bonfire_activity_time = 104;  //所参加的篝火活动开始时间
	required uint32 partner_today_junior_recurit_count = 105;  //今日JUNIOR伙伴招募次数
	required uint32 partner_today_junior_recurit_cd = 106;  //今日JUNIOR伙伴招募cd
	required uint32 partner_today_senior_recurit_count = 107;  //今日senior伙伴招募次数
	required uint32 partner_today_senior_recurit_cd = 108;  //今日senior伙伴招募cd
	optional DBPlayerGuildChuanGongInfo guild_chuan_gong_info = 109; //帮会传功信息
	required uint32 partner_recruit_junior_free_count = 110; //低级招募免费次数
	required uint32 partner_recruit_senior_free_count = 111; //高级招募免费次数
	required uint32 silver = 112; //银币
	repeated DBPlayerJiuGongBaGuaInfo jiu_gong_ba_gua_reward_info = 113; //九宫八卦领奖信息

	required uint32 default_hair = 114;
	required uint32 default_icon = 115;
	required uint32 default_hair_color = 116;
	required uint32 default_up_color = 117;
	required uint32 default_down_color = 118;
}

message DBAutoAddHp
{
	required uint32 open_auto_add_hp = 1; //0表示关闭自动补血，1表示开启
	required uint32 auto_add_hp_item_id = 2; //自动补血的道具ID
	required uint32 auto_add_hp_percent = 3; //自动补血的百分比，0-100
	required uint32 hp_pool_num = 4;   //剩余的血池容量	
}

message ChengjieTaskDb
{
	required uint64 playerid = 1;
	required uint64 investor = 2;
	required uint32 fail = 3; //失败次数
	required uint32 shuangjin = 4; //
	required uint32 exp = 5; //
	required uint32 courage = 6;
	required uint32 taskid = 7;
	required uint64 cd = 8; //任务消失剩余时间
	required uint64 complete_cd = 9; //完成任务剩余时间 >0:已接
	required bool complete = 10; //true 任务已完成
	required uint64 accept_cd = 11; //可接任务剩余时间 0:可接
	required uint64 accepter = 12;
	required bool anonymous = 13; //
	required string declaration = 14; //
	required uint32 step = 15; //悬赏价格段位
}
message LoadChengjieTask
{
	repeated ChengjieTaskDb data = 1;
}

message DBServerLevel
{
	required uint32 level_id = 1;
	required uint32 break_goal = 2;
	required uint32 break_num = 3;
	repeated uint64 break_reward = 4;
}

message DBHreoChallengeInfo
{
	required uint32 id = 1;      //英雄id
	required uint32 star = 2;	 //历史最高星级
	required uint32 reward_flag = 3; //是否有未领取奖励标志 1:是
	repeated DBHreoChallengeItemInfo item_info = 4; //奖励物品信息
}

message DBHreoChallengeItemInfo
{
	required uint32 item_id = 1;    //物品id
	required uint32 item_num = 2;	//物品数量
}

message DBPlayerLevelRewardInfo
{
	required uint32 id = 1;    //等级奖励表id
	required bool receive = 2; //奖励是否领取
}

message DBPlayerOnlineRewardInfo
{
	required uint32 befor_online_time = 1;   //今日在线总时长
	required uint32 use_reward_num = 2;      //今日已领奖次数
	required uint32 reward_num = 3;          //今日达到领奖要求的剩余未领奖次数
	repeated uint32 reward_table_id = 4;     //已经领取的奖励表id
	repeated uint32 reward_id_today = 5;     //今日可领奖励表id
}

message DBPlayerSignInEveryDayInfo
{
	required bool today_sign = 1;       //记录今日是否已经签到
	required uint32 cur_month = 2;      //当前月份
	required uint32 month_sum = 3;      //本月累计签到次数
	required uint32 yilou_num = 4;      //本月遗漏次数
	required uint32 buqian_sum = 5;     //本月累计可补签次数
	required uint32 activity_sum = 6;   //本月通过活跃度获取的次数
	repeated DBPlayerSignInLeiJiReward leiji_reward = 7;  //本月累计奖励信息
}

message DBPlayerSignInLeiJiReward
{
	required uint32 id = 1;              //奖励表id
	required uint32 status = 2;			 //奖励状态
}

//奖励找回信息
message DBPlayerActiveRewardBackInfo
{
	required uint32 id = 1;   //找回表id
	required uint32 num = 2;  //可找回奖励数量
	required uint32 normo_exp = 3; //普通找回经验
	required uint32 perfect_exp = 4; //完美找回经验
	required uint32 normo_coin = 5; //普通找回银两
	required uint32 perfect_coin = 6; //完美找回银两
	required uint32 normo_use = 7;   //普通找回消耗的银票
	required uint32 perfect_use = 8; //完美找回消耗的金票
}

//摇钱树信息
message DBPlayerYaoQianShuInfo
{
	required uint32 sum_num = 1;   //今日可摇总次数
	required uint32 use_num = 2;   //今日已经摇了多少次
	required uint32 free_num = 3;  //今日剩余免费次数
	required uint32 next_need_money = 4; //下一次摇需要消耗的元宝
	required uint32 cur_jizhi_num = 5; //当前机制数(其实就是获得两倍以上暴击后又摇了多少次) 
	required uint32 cur_suiji_zhong = 6;  //当前的随机种子
	required uint32 beilv_num = 7;      //当 cur_jizhi_num == cur_suiji_zhong时能获得的暴击倍率
}

//登录奖励信息
message DBPlayerLoginRewardInfo
{
	required bool open = 1; //功能是否开启
	required bool receive_time = 2; //是否在领取时间内
	required uint32 open_time = 3; //功能开启的时间点
	required uint32 login_day = 4; //当前是此功能开启后登录的第几天
	repeated DBPlayerLoginReceiveInfo info = 5; //奖励领取情况
}

//奖励领取信息
message DBPlayerLoginReceiveInfo
{
	required uint32 id = 1; //奖励表id
	required uint32 statu = 2; //领取状态
}

//赐福奖励信息
message DBPlayerCifuRewardInfo
{
	required uint32 id = 1;  //赐福表id
	required uint64 time = 2; //上次领取时间
}

//地宫修炼信息
message DBPlayerDiGongXiuLianInfo
{
	required uint32 digong_table_id = 1; //地宫修炼表索引id
	required uint32 digong_task_id = 2; //地宫试炼任务当前身上已接未提交任务id
	required uint32 digong_time_statu = 3; //地宫试炼任务时间标志0:今日接 1:昨日接的
	required uint32 digong_reward_beilv = 4; //地宫试炼任务当前奖励倍率
	required uint32 digong_huan_num = 5; //地宫试炼任务今日已完成总环数
	required uint32 digong_lun_num = 6;  //地宫试炼任务今日已完成总轮数

}

message DBPlayerGuildChuanGongInfo
{
	required uint32 bei_chuan_num = 1; //被动传功次数
	required uint32 zhu_chuan_num = 2; //主动传功次数
}

//九宫八卦领奖信息
message DBPlayerJiuGongBaGuaInfo
{
	required uint32 id = 1; //奖励id
	required uint32 task_id = 2; //对应的任务id
	required uint32 statu = 3; //奖励领取状态
}
